site/docs/0.0/content/font.md

```diff
@@ -0,0 +1,746 @@
+---
+layout: docs
+title: WebFont Proxima Nova
+description: The Proxima Nova font suite is available for official use by University of Arizona employees through a license with Adobe.
+group: content
+redirect_from: "/docs/4.3/content/"
+toc: true
+---
+
+## Centrally managed typekit webfont
+
+For ease of integration into web projects around campus, the Arizona Digital
+team manages a typekit webfont project that is referenced in Arizona Bootstrap.
+
+#### Notes to consider when using the centrally mananged webfont project.
+Our license with typekit allows anyone with a netid to create their own webfont
+project in creative cloud typekit and use that in all of their web projects. Of
+course there are pluses and minus to each approach.
+
+#### Managing your own project
+
+**Pluses**
```

Comment:
Nitpicky/style suggestion: Maybe "Advantages" / "Disadvantages" instead of "Pluses" / "Minuses"?

Reply:
Changed to "Advantages" / "Disadvantages"